packages/oas-utils/src/helpers/httpMethods.ts

```diff
@@ -1,58 +1,60 @@
 import type { RequestMethod } from '@/entities/spec/requests'
 
-export const REQUEST_METHODS: {
-  [x in RequestMethod]: {
-    short: string
-    color: string
+/** HTTP methods in a specific order */
+const ORDERED_METHODS: RequestMethod[] = [
```

Comment:
I would just order these in entitites/spec/requests and put on a comment that they should not be re-ordered. Then we only have one source of truth